ci-operator/step-registry/fips-check/fips-or-die/fips-check-fips-or-die-commands.sh

```diff
@@ -111,6 +111,22 @@ images=("quay.io/openshift-qe-optional-operators/myapp:v1.16-1"
 set_proxy
 run_command "oc whoami"
 run_command "oc version -o yaml"
+
+version=`oc get clusterversion version -o=jsonpath="{.status.desired.version}"`
+if [[ $version =~ "multi" ]]; then
+    echo "This is a multi-arch cluster, skip it."
+    exit 0
+fi
+
+run_command "oc extract secret/pull-secret -n openshift-config --confirm --to /tmp"; ret=$?
+if [[ $ret -eq 0 ]]; then
+    auths=`cat /tmp/.dockerconfigjson`
+    if [[ $auths =~ "5000" ]]; then
+        echo "This is a disconnected env, skip it."
+        exit 0
+    fi
+fi
+
 node_name=`oc get node -l node-role.kubernetes.io/master= -o=jsonpath="{.items[0].metadata.name}"`
 arch=`oc get node/$node_name -o=jsonpath="{.status.nodeInfo.architecture}"`
 if [[ $arch != "amd64" ]]; then
```

Comment:
Beyond ARM64, are the other multi-arch folks aware of this skip?

/cc @Prashanth684 @andymcc @werled @prb112

Reply:
/hold until reviews

Reply:
I think the `FIPS or die` test on `AMD64` is enough, @mrunalp What do you think? Thanks!

Reply:
cc @jaypoulz FIPS mode is supported on P/Z 4.12+